cleancode/src/main/java/com/epam/cleancode/comments/MortgageInstallmentCalculator.java

```diff
@@ -1,39 +1,35 @@
 package com.epam.cleancode.comments;
 
-public class MortgageInstallmentCalculator {
+
+class MortgageInstallmentCalculator {
+
+    private static final String INVALID_INPUT_EXCEPTION = "Negative values are not allowed";
+    private static final double NUMBER_FOR_CONVERT_TO_DECIMAL = 100.0;
+    private static final double AMOUNT_OF_MONTHS = 12.0;
```

Comment:
amount of what? as I understand in year, but in code where this constant will be used it is not clear.